### PR TITLE
No need to use execinfo in KWSys in vtk project

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

File: projects/vtk.cmake [hunk 1]
```diff
@@ -18,6 +18,7 @@ superbuild_add_project(vtk
   DEPENDS_OPTIONAL ospray ${vtk_platform_dependencies}
   CMAKE_ARGS
     -DCMAKE_MACOSX_RPATH=OFF
+    -DKWSYS_NO_EXECINFO=ON
```
Comment (Meakk) [inline projects/vtk.cmake:21]: Is it fixing the issue in netcdf?

Comment (mwestphal) [reply projects/vtk.cmake:21]: there is no issue in netcdf

Comment (mwestphal) [reply projects/vtk.cmake:21]: see this discussion: https://github.com/f3d-app/f3d-superbuild/pull/83/files#r1257313491

**Comment by mwestphal** [comment]
not required for building on alpine, closing

**Pull request closed without merging**